requirements.txt

```
@@ -4,3 +4,4 @@ numpy>=1.19.5
 Pillow>=8.2.0
 python-magic>=0.4.22
 regex>=2021.4.4
+Wand==0.6.7
```

```suggestion
Wand>=0.6.7
```

Unless we really need specifically `Wand==0.6.7`, it's more flexible to include requirements as minimum, not pinned versions.